src/jet/main.clj

```diff
@@ -13,7 +13,7 @@
 
 (set! *warn-on-reflection* true)
 
-(defn- parse-opts [options]
+(defn parse-opts [options]
```

Comment:
Made this function public in order to test it, because the warnings prevented me from calling it in tests.

Reply:
You can test private functions like this: `(#'parse-opts ...)`

Reply:
True. But I've tended to shun the usage of private functions since I see them as something that comes from the object oriented world and don't see the need for this "hiding".

If you want me to revert this, let me know. 

Thanks

Reply:
Prive in Clojure isn't about hiding, since you can access the var anyway, but about public API: what you're supposed to use and what is implementation detail (may have breaking changes in the future).

Reply:
> Private in Clojure isn't about hiding, since you can access the var anyway, but about public API: what you're supposed to use and what is implementation detail (may have breaking changes in the future).

OK, but do you really think that somebody will include this project just to require `parse-opts`?